test/unit/crate_name/crate_name_test.bzl

```diff
@@ -71,6 +71,17 @@ def _slib_library_name_test_impl(ctx):
     assert_argv_contains(env, tut.actions[0], "--codegen=extra-filename=-2102077805")
     return analysistest.end(env)
 
+def _no_extra_filename_test_impl(ctx):
+    """Check that no extra filename is used.
+
+    Args:
+      ctx: rule context.
+    """
+    env = analysistest.begin(ctx)
+    tut = analysistest.target_under_test(env)
+    assert_argv_contains(env, tut.actions[0], "--codegen=extra-filename=")
```

Comment:
WDYT about removing --codegen=extra-filename= from the command line altogether?

Reply:
good idea, let me try